common/lvm/util.go

```diff
@@ -128,3 +129,27 @@ func (lvmNodesDevicePvVgConfig *LvmNodesDevicePvVgConfig) ConfigureLvmNodesWithD
 	}
 	return nil
 }
+
+func SetupLvmNodes(vgName string, size int64) (LvmNodesDevicePvVgConfig, error) {
+	var lvmNodeConfig LvmNodesDevicePvVgConfig
+	loopDevice := e2e_agent.LoopDevice{
+		Size:   size,
+		ImgDir: "/tmp",
+	}
+	workerNodes, err := ListLvmNode(common.NSOpenEBS())
+	if err != nil {
+		return lvmNodeConfig, fmt.Errorf("failed to list zfs worker nodes, error: %v", err)
+	}
+
+	lvmNodeConfig = LvmNodesDevicePvVgConfig{
+		VgName:        vgName,
+		NodeDeviceMap: make(map[string]e2e_agent.LoopDevice), // Properly initialize the map
+	}
+	for _, node := range workerNodes {
+		lvmNodeConfig.NodeDeviceMap[node] = loopDevice
+	}
+
+	logf.Log.Info("setup node with loop device and zfs pool", "zfs node config", lvmNodeConfig)
```

Comment:
```suggestion
	logf.Log.Info("setup node with loop device and lvm pool", "lvm node config", lvmNodeConfig)
```

Reply:
updated the log https://github.com/openebs/openebs-e2e/compare/aac2d9ae2cec3b80cd89fee4cb1ff57ddc44c215..61f53a539d78cb3e47dd205a316b7c0abc9840b0